README.rst

```diff
@@ -144,6 +144,22 @@ argument to ``bin/crawl-foo-org``. The crawler will then only fetch and index
 that specific URL.
 
 
+Slack-Notifications
+-------------------
+
+The ``ftw.crawler`` supports Slack-Notifications. Those notifications can be used
+to monitor the crawler on possible errors while crawling.
+To enable slack-notifications for your environment, you need to do the following things:
+
+- Make sure `slacker <https://github.com/os/slacker>`_ is installed in your env.
+- Set the `SLACK_TOKEN` and the `SLACK_CHANNEL` params in your crawler config or
+- use the `--slacktoken` and the `--slackchannel` arguments in the command line when
+  calling the `/crawl` script.
+
+To generate your Slack-Token you can visit the Slack-Homepage and search for it in the
```

Comment:
Where exactly can I find the tokens?
Be aware that the tokens at https://api.slack.com/docs/oauth-test-tokens are only test tokens and should not be used in production..